### PR TITLE
[Spreadsheet] Use DontUseNativeColorDialog parameter when using QtCol…

### PR DESCRIPTION
**Pull request by mwganson**
…orDialog::getColor()

Similar PR for TechDraw workbench:

https://github.com/FreeCAD/FreeCAD/pull/5322

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5326">pipeline status</a> for feature branch [PR_5326](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5326). Pipeline [440711183](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/440711183) was triggered at [1eed45f](https://github.com/FreeCAD/FreeCAD/pull/5326/commits/1eed45f2552cb64b8ea2d8b2042095ac53c4e826). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Spreadsheet/Gui/qtcolorpicker.cpp [hunk 1]
```diff
@@ -902,7 +902,12 @@ void ColorPickerPopup::getColorFromDialog()
 {
     //bool ok;
     //QRgb rgb = QColorDialog::getRgba(lastSel.rgba(), &ok, parentWidget());
-    QColor col = QColorDialog::getColor(lastSel,parentWidget(),0,QColorDialog::ShowAlphaChannel);
+    QColor col;
+    if (Gui::DialogOptions::dontUseNativeColorDialog()){
+        col = QColorDialog::getColor(lastSel,parentWidget(),0,QColorDialog::ShowAlphaChannel|QColorDialog::DontUseNativeDialog);
```
Comment (donovaly) [inline src/Mod/Spreadsheet/Gui/qtcolorpicker.cpp:907]: Minor comment: we use spaces behind commas

I will merge and fix the style later today.

Comment (donovaly) [reply src/Mod/Spreadsheet/Gui/qtcolorpicker.cpp]: seems as admin I can change this directly in the PR

**Comment by mwganson** [comment]
Okay.  Thanks.  I just copied and pasted the original line and added what I needed to add.

**Pull request merged**